src/core/operator_list.js

```diff
@@ -780,6 +780,13 @@ class OperatorList {
             transfers.push(data.buffer, minMax.buffer);
           }
           break;
+        case OPS.paintFormXObjectBegin:
+          const [matrix, bbox] = argsArray[i];
+          transfers.push(matrix.buffer);
```

Comment:
Given the previous comment.
```suggestion
          if (matrix) {
            transfers.push(matrix.buffer);
          }
```